objects/rct2ww/scenery_large/rct2.ww.pagodatw.json

```diff
@@ -92,7 +92,7 @@
             "pl-PL": "Dach pagody",
             "ru-RU": "Пагода",
             "pt-BR": "Torre Pagode",
-            "ja-JP": "パゴダ"
+            "ja-JP": "パゴダタワー"
```

Comment:
@AaronVanGeffen

Reply:
He's changed it into "pagoda tower". That sounds like a pleonasm to me.